account_reconcile_restrict_partner_mismatch/tests/test_reconciliation.py

```diff
@@ -0,0 +1,94 @@
+from odoo.addons.account.tests.account_test_classes import AccountingTestCase
+from odoo.exceptions import UserError
+
+
+class TestReconciliation(AccountingTestCase):
+
+    def setUp(self):
+        super(TestReconciliation, self).setUp()
+        self.env = self.env(context=dict(
+            self.env.context, tracking_disable=True, specific_tests=True)
```

Comment:
```suggestion
            self.env.context, tracking_disable=True, test_partner_mismatch=True)
```
Please use a more 'specific' context key :wink: